store/helper/helper.go

```diff
@@ -1001,7 +1001,9 @@ func (h *Helper) GetPDRegionStats(tableID int64, stats *PDRegionStats, noIndexSt
 			logutil.BgLogger().Error("err", zap.Error(err))
 		}
 	}()
-
+	if resp.StatusCode != http.StatusOK {
+		return errors.New("GetPDRegionStats returns error")
+	}
```

Comment:
Just a suggestion, we can read the body to returrn more verbose error.

Reply:
I have a plan to implement a better HTTP framework to unite the usage like these cases.

Reply:
I maybe improve this for this PR.

Reply:
@xhebox PTAL